Docker/OpenWRT/Dockerfile

```diff
@@ -0,0 +1,18 @@
+# This Dockerfile is used in conjunction with the docker-OpenWRT.yml GitHub Action.
+FROM alpine:latest AS builder
+
+RUN apk add argp-standalone asciidoc bash bc binutils bzip2 cdrkit coreutils \
+  diffutils elfutils-dev findutils flex musl-fts-dev g++ gawk gcc gettext git \
+  grep intltool libxslt linux-headers make musl-libintl musl-obstack-dev \
+  ncurses-dev openssl-dev patch perl python3-dev rsync tar \
+  unzip util-linux wget zlib-dev autoconf automake libtool
+COPY . /workspace
+RUN cd /workspace && ./autogen.sh && ./configure --enable-all && make
+
+FROM openwrt/rootfs:x86-64-22.03.0
```

Comment:
`22.03.0` is the first release in this stable series from [9/2022](https://openwrt.org/releases/22.03/notes-22.03.0) so little bit outdated, I would try to use current latest `22.03.3` release. And in this case I would probably recommend to stick using a snapshot build `openwrt/rootfs:x86-64-22.03-SNAPSHOT` which would later become next OpenWrt release in the `22.03.y` series, so `22.03.4`.

Runtime overview of the versions.

```
$ for release in 21.02.0 21.02.5 22.03.0 22.03.3 21.02-SNAPSHOT 22.03-SNAPSHOT snapshot; do \
       echo x86-64-${release} $(docker run --rm --pull always openwrt/rootfs:x86-64-$release sh -c 'ls /usr/lib/libwolfssl*') ; \
  done 2>&1 | grep wolfssl
x86-64-21.02.0 /usr/lib/libwolfssl.so.4.7.0.66253b90
x86-64-21.02.5 /usr/lib/libwolfssl.so.5.5.1.99a5b54a
x86-64-22.03.0 /usr/lib/libwolfssl.so.5.4.0.ee39414e
x86-64-22.03.3 /usr/lib/libwolfssl.so.5.5.4.ee39414e
x86-64-21.02-SNAPSHOT /usr/lib/libwolfssl.so.5.5.3.99a5b54a
x86-64-22.03-SNAPSHOT /usr/lib/libwolfssl.so.5.5.4.ee39414e
x86-64-snapshot /usr/lib/libwolfssl.so.5.5.4.e624513f
```

Reply:
I ended up with a single reference since I was using the COPY command to overwrite the existing libwolfssl and extracting out the exact version hash seemed painful to be testing out the API/ABI. Though I can rework some of the scripting to try to be version agnostic.